icubam/messaging/telegram/integrator.py

```diff
@@ -0,0 +1,63 @@
+import tornado.queues
+from icubam.messaging.telegram import bot, updater, webhook
+
+
+class TelegramSetup:
+  """Sets up the telegram in the server support depending on the config."""
+  def __init__(self, config, db, scheduler=None, tg_bot=None):
+    self.config = config
+    self.queue = tornado.queues.Queue()
+    self.bot = None
+    self.processor = None
+    self.fetcher = None
+    if self.is_on:
+      self.bot = bot.TelegramBot(config) if tg_bot is None else tg_bot
+      self.processor = updater.UpdateProcessor(
+        config, db, self.queue, scheduler, tg_bot=self.bot
+      )
+      self.fetcher = updater.UpdateFetcher(config, self.queue, tg_bot=self.bot)
+
+  @property
+  def uses_webhook(self):
+    url = self.config.server.base_url
+    return 'https' in url and 'localhost' not in url
```

Comment:
Could you add an inline comment explaining this? It could have been 127.0.0.1 as well FWIW.

Reply:
good one. The telegram webhook can only be set on a https route. when you are in dev mode on your machine you usually dont have this.